biodatacatalyst-ui/src/main/webapp/picsureui/search-interface/filter-model.js

```diff
@@ -71,12 +71,12 @@ define(["backbone", "handlebars", "picSure/settings", "picSure/queryBuilder", "o
                     category: this.generateVariableCategory(searchResult),
                     min: min,
                     max: max,
-                    filterType: min===undefined ? "lessThan" : max===undefined ? "greaterThan" : "between",
+                    filterType: min=="" ? "lessThan" : max=="" ? "greaterThan" : "between",
                     isHarmonized: searchUtil.isStudyHarmonized(searchResult.result.metadata.columnmeta_study_id),
                     topmed: searchResult.result.varId.includes('phv'),
                 });
                     tagFilterModel.requireTag(searchResult.result.metadata.columnmeta_study_id);
-                    this.addExportColumn(searchResult, 'filter', undefined, undefined, min, max);
+                    this.addExportColumn(searchResult, 'filter', undefined, undefined, min == "" ? searchResult.result.metadata.columnmeta_min : min, max == "" ? searchResult.result.metadata.columnmeta_max : max);
```

Comment:
it's me, your linter
```suggestion
                    this.addExportColumn(
                        searchResult, 'filter', undefined, undefined,
                        min == "" ? searchResult.result.metadata.columnmeta_min : min,
                        max == "" ? searchResult.result.metadata.columnmeta_max : max
                    );
```